docs/modules/commons/pages/cli.adoc

```diff
@@ -333,3 +333,109 @@ applitools.app-name=name
 web.driver.chrome.experimental-options={"prefs": {"profile": {"cookie_controls_mode": 0}}}
 batch-6.story.execution-timeout=PT15M
 ----
+
+== Find and fix formatting issues (Spotless checks)
+
+Spotless checks refer to code formatting and linting validations performed by the https://github.com/diffplug/spotless[Spotless plugin], a tool designed to enforce consistent code quality standards. Spotless automates the formatting of code based on predefined rules and integrates with linting tools for multiple programming languages, such as Java, Kotlin, Groovy, and more.
+
+Spotless offers a number of generic formatters (indentation correction, normalize line endings, enforce consistent EOF files, enforce file encoding, normalize quotes, remove debugging or unnecessary comments, sort imports/dependencies, etc.) that can help enforce consistent code hygiene in your project. These formatting rules are versatile and can be applied to any type of file, ensuring your codebase is clean and professional.
+
+=== Benefits of using Spotless
+
+* Enforces Consistency: Guarantees uniform code formatting across individual developers or teams.
+* Saves Time: Eliminates manual effort by automating formatting tasks.
+* CI/CD Integration: Seamlessly integrates with CI/CD pipelines to ensure formatting compliance before deployment.
+* Broad Language Support: Works with various languages and styles, making it highly versatile.
+
+By integrating Spotless into the `build` process or CI pipeline, teams can maintain a clean and consistent codebase while catching issues before merging changes into the main branch.
+
+=== Key Spotless tasks
+
+The task validates that all files comply with the specified formatting rules.
```

Comment:
add an information this task is executed automatically and doesn't require explicit invocation